infrastructure_source_code.md

```diff
@@ -0,0 +1,44 @@
+# DataWorks Open Source Infrastructure Source Code Policy
+
+## Purpose
+
+This document sets out local policy, for the Department for Work and Pensions (DWP) DataWorks team, on what types of information can and cannot be included in infrastructure code that is made publicly available.
+
+It is expected that, at some point, this document will be superceded by a DWP-wide open-source infrastructure code policy which encompasses all DWP public GitHub repositories. Some useful [DWP guidance on creating open source products](https://confluence.service.dwpcloud.uk/pages/viewpage.action?title=Guidance+on+creating+open+source+products&spaceKey=EN) currently exists, but it has not yet been made open-source.
+
+## Scope
+
+This policy covers infrastructure code. Please refer to the [index](README.md#index) for links to other related polices.
+
+## General Approach
```

Comment:
We also need to worry about things like terraform state files which will by necessity have sensitive information within them which cannot be easily obfuscated. Although not technically in the scope of this document (and/or covered by the general provisions below) we should probably provide a specific steer on these.